### ISSUE TITLE
About Qualitative & Quantitative Comparison

### ISSUE THREAD

**Issue by haeyun-choi** [taekkii/ExBluRF#2]
First of all, thank you for the great work.

I'm leaving an issue because I want to ask a question about the pretraining weights you used 
when comparing performance with other works. 

What weights are used when utilizing NAFNet and Restormer? (e.g. NAFNet-32 trained with GoPro, NAFNet-64 trained with REDS)

I am asking because, for a fair comparison, both models should not be trained on test data. (every scene)
I think there is no way but to use the pre-trained deblurring networks.

Thank you again.

**Comment by taekkii** [comment]
Hi, thank you for the question.

For NAFNet, we have used NAFNet-64 trained on REDS, as it is the very weight provided in quickstart on their Github repo.
For Restormer, we also used the quickstart (demo) weight, and I believe the weight is trained on GoPRO according to the paper.
We do not fine-tune the weight with our scene data for the very reason you have pointed out (fairness purpose).

I believe the performance of both models could be improved by a little if they're trained on RealBlur dataset because RealBlur shares the type of camera as ExBlur scene. Nevertheless, I think neither of them could beat any of the 3D-reconstruction-with-blur methods (DeblurNeRF and its variants including ours) since they are built for the specific task, not only for single-image-deblurring.
